index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
+        <div class="header_menu">
+          <img class="menu_icon" src="assets/ic_menu.png" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="headline_title">
+        <span class="sub-headline_name">Hey there, I'm Arslan</span>
+        <span class="sub-headline">I'm a software developer</span>
```

Comment:
- [ ] **Required:** These two `span` elements need to be their own `h1` and `h2` tag because it is a bad SEO practice to have additional elements wrapped inside of an `h1` tag